docs/pages/datapoint.js

```diff
@@ -39,9 +39,70 @@ export default function DatapointPage({ generatedDocGen }: {| generatedDocGen: D
           />
         </MainSection.Subsection>
       </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Whenever possible, use the full number with separators to ensure clarity across all languages."
+            defaultCode={`
+<Datapoint size="lg" title="Total impressions" value="1,451" trend={{value: 10.1, accessibilityLabel: "Trending up"}}  />
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use long decimal values for trend. Limit the trend to one decimal point."
+            defaultCode={`
+<Datapoint size="lg" title="Saves" value="10,392" trend={{value: -12.193, accessibilityLabel: "Trending down"}} />
+`}
+          />
+          {/*
+            The example below should also display a localized trend value, but given trend accepts only a number,
+            the value cannot be localized. Once the API is changed, we should add a localized trend to this example.
+            */}
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Make sure to localize numeric values."
+            defaultCode={`
+<Datapoint size="lg" title="Leistung" value="3.000,25" />
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use subjective values for Datapoint’s value."
+            defaultCode={`
+<Datapoint size="lg" title="Performance" value="Bad" />
+
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Be certain to use a positive `trendSentiment` only when the trend is overtly positive for the end user. Use a neutral `trendSentiment` in cases of ambiguity."
+            defaultCode={`
+<Datapoint size="lg" title="Total spend" value="$14,325" trend={{value: 5.6, accessibilityLabel: "Trending up"}} trendSentiment="neutral"  />
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use Status' `subText` to display extraneous messaging."
```

Comment:
@ponori Does this one belong here? It's Status not Datapoint, how does it relate to Datapoint?

Reply:
Thanks! Fixed!